packages/x-components/src/components/animations/animate-width.vue

```diff
@@ -1,5 +1,6 @@
 <template>
-  <transition v-on="$listeners" name="x-animate-width-" v-bind="$attrs">
+  <!-- eslint-disable-next-line vue/attributes-order -->
+  <transition v-bind="$attrs" v-on="$listeners" name="x-animate-width-">
```

Comment:
I'm just curious, why the attributes order needs to change here?

Reply:
While Vue 2 doesn't care about props merge when using v-bind in Vue 3 the order is important: https://v3-migration.vuejs.org/breaking-changes/v-bind

To migrate without a warning we should put the v-bind always first. In this specific case we will also remove the v-on once we migrate to Vue 3 as it is included in the new v-bind: https://v3-migration.vuejs.org/breaking-changes/v-on-native-modifier-removed.html